docker-compose.yml

```diff
@@ -10,14 +10,7 @@ services:
             - ./docs:/srv/jekyll
         ports:
             - "4000:4000"
-    web:
```

Comment:
@emkap01 can you remove this file from the pull request? We modified this version to get it to work locally for you, but don't want that in everyone's version. Here's instructions: http://stackoverflow.com/questions/12481639/remove-files-from-git-commit

If you're not comfortable doing this, we can do it in person on Tuesday instead.